Fakebook.Posts/Fakebook.Posts.DataAccess/Models/FakebookPostsContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Fakebook.Posts.DataAccess.Models {
+    public partial class FakebookPostsContext : DbContext {
+        public FakebookPostsContext(DbContextOptions<FakebookPostsContext> options) : base(options) { }
+        public virtual DbSet<Post> Posts { get; set; }
+        public virtual DbSet<Comment> Comments { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder) {
+            modelBuilder.Entity<Post>(entity => {
+                entity.ToTable("Post");
```

Comment:
`entity.ToTable("Post", "Fakebook");`
`entity.ToTable("Comment", "Fakebook");`
would create a specific schema, shouldn't we do this?

Reply:
schema has now been specified.